docs/user/airbrakes.rst

```diff
@@ -0,0 +1,402 @@
+    second.
+  - ``v_y``: The y component of the velocity of the rocket, in meters per 
+    second.
+  - ``v_z``: The z component of the velocity of the rocket, in meters per 
+    second.
+  - ``e0``: The first component of the quaternion representing the rotation 
+    of the rocket.
+  - ``e1``: The second component of the quaternion representing the rotation 
+    of the rocket.
+  - ``e2``: The third component of the quaternion representing the rotation 
+    of the rocket.
+  - ``e3``: The fourth component of the quaternion representing the rotation 
+    of the rocket.
+  - ``w_x``: The x component of the angular velocity of the rocket, in 
+    radians per second.
+  - ``w_y``: The y component of the angular velocity of the rocket, in 
+    radians per second.
+  - ``w_z``: The z component of the angular velocity of the rocket, in 
+    radians per second.
+- ``state_history``: A list containing the state of the simulation at every 
+  time step up to the current time step. The state of the simulation at the 
+  previous time step is the last element of the list.
+- ``observed_variables``: A list containing the variables of interest returned
+  by the controller function at every time step up to the current time step.
+  The variables of interest returned by the controller function at the previous
+  time step is the last element of the list.
+- ``air_brakes``: The ``AirBrakes`` instance being controlled.
+    
+Our example ``controller_function`` will deploy the air brakes when the rocket
+reaches 1500 meters above the ground. The deployed level will be function of the
+vertical velocity at the current time step and of the vertical velocity at the
+previous time step.
+
+Also, the controller function will check for the burnout of the rocket's motor 
+and only deploy the air brakes if the rocket has reached burnout. 
+
+Then a limitation for the opening speed of the air brakes will be set. The air 
+brakes will not be able to deploy faster than 0.2 percentage per second.
+
+Lets define the controller function:
+
+.. jupyter-execute::
+
+    def controller_function(
+        time, sampling_rate, state, state_history, observed_variables, air_brakes
+    ):
+        # state = [x, y, z, vx, vy, vz, e0, e1, e2, e3, wx, wy, wz]
+        altitude_ASL = state[2]
+        altitude_AGL = altitude_ASL - env.elevation
+        vx, vy, vz = state[3], state[4], state[5]
+        mach_number = (vx**2 + vy**2 + vz**2) ** 0.5 / env.speed_of_sound(
+            altitude_AGL - env.elevation
+        )
+
+        # Get previous state from state_history
+        previous_state = state_history[-1]
+        previous_vz = previous_state[5]
+
+        # If we wanted to we could get the returned values from observed_variables:
+        # returned_time, deployed_level, drag_coefficient = observed_variables[-1]
+
+        # Check if the rocket has reached burnout
+        if time > Pro75M1670.burn_out_time:
```

Comment:
You have a good point here. This is actually a pretty big problem that we have overlooked.

Is there any way to nicely solve this, without making significant changes to the `Flight` class?

We could just add some extra calculations to get the accelerations, in case there are air brakes are in the simulation. That is the only way I can think of right now